### DISCUSSION TITLE
glTFast package not automatically installing with the other packages

### DISCUSSION THREAD

**Discussion by gulnuravci** [readyplayerme/rpm-unity-sdk-core#80] [Q&A]
Hi,

So I'm trying to get started with ReadyPlayerMe in Unity. According to Youtube videos and one on the official channel that was uploaded 10 days ago, when you copy "https://github.com/readyplayerme/rpm-unity-sdk-core.git" to "add package from git URL" in Unity, it seems to install glTFast, Ready Player Me Avatar Loader, Ready Player Me Core, and Ready Player Me WebView packages for others. For me, glTFast seems to be missing from my package manager, and other attempts to install it separately have not resolved the errors that come up. My Unity version is 2021.3.15f1, and I tried the same method in a completely new project and had the same issue. What could I be missing? Thank you!

**Comment by PabloCirre** [comment]
me too :( solutions?

**Reply by rYuuk** [reply]
@branvan3000 This should have been fixed, which version are you using? And are you getting an error?

**Reply by rYuuk** [reply]
There shouldn't be any of these scripting define symbols in project settings before installing the ready player me package.
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/1121080/280cff67-edfb-4c0c-a9d0-4f5f0b246a06)
